frontend/src/services/Native/index.ts

```diff
@@ -1,16 +1,19 @@
+import { systemClient } from '../System';
 import { NativeRobosatsPromise, NativeWebViewMessage, NativeWebViewMessageSystem } from './index.d';
 
 class NativeRobosats {
-  constructor() {
-    this.messageCounter = 0;
-  }
-
   public torDaemonStatus = 'NOTINIT';
 
-  private messageCounter: number;
+  private messageCounter: number = 0;
 
   private pendingMessages: { [id: number]: NativeRobosatsPromise } = {};
 
+  public cookies: { [key: string]: string } = {};
```

Comment:
Storing the cookies on the client will avoid implementing async functions to obtain cookies, to avoid inconsistencies this object is only modified by React-Native directly.